### PR TITLE
Hazelcast Adpater added, closes #3973

### PR DESCRIPTION
**Pull request by kaggrwal**
Hazelcast Adpater added, closes [#issue](https://github.com/appwrite/appwrite/issues/3973)

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -15,6 +15,7 @@
             <file>./tests/Cache/NoneTest.php</file>
             <file>./tests/Cache/RedisTest.php</file>
             <file>./tests/Cache/ShardingTest.php</file>
+            <file>./tests/Cache/HazelCastTest.php</file>
```
Comment (stnguyen90) [inline phpunit.xml:18]: Incorrect filename

File: tests/Cache/HazelCastTest.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Utopia\Tests;
+
+use Memcached as Memcached;
+use Utopia\Cache\Cache;
+use Utopia\Cache\Adapter\Hazelcast as HazelCastAdapter;
```
Comment (stnguyen90) [inline tests/Cache/HazelCastTest.php:7]: No need to rename

File: tests/Cache/HazelCastTest.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+use Utopia\Tests\Base;
+
+class HazelCastTest extends Base
```
Comment (stnguyen90) [inline tests/Cache/HazelCastTest.php:10]: This and hte filename should be HazelcastTest

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out. 

Would you please also fix the lining errors?

**Comment by kaggrwal** [comment]
Hi @stnguyen90, I'm getting no issues when running lint on my local. I have attached the screenshot also.
Screenshot from 2022-10-30 21-38-59

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
> Hi @stnguyen90, I'm getting no issues when running lint on my local. I have attached the screenshot also.

@kaggrwal, running `./vendor/bin/pint` will actually fix your code so that's why running `./vendor/bin/pint --test` after passes. You probably have code you haven't committed or pushed.

**Review by stnguyen90 — APPROVED** [review]
Awesome work on your PR! We've approved your work and it'll be merged soon!

**Comment by kaggrwal** [comment]
@stnguyen90 Please merge it, as it's been pending since long time.

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**